client/apps/edit/components/content2/index.coffee

```diff
@@ -13,6 +13,8 @@ module.exports = React.createClass
       @props.article.save()
       @onChange()
     ), 800)
+
+    # FIXME: Polling autosave for `last_updated`, not sure where though
     @props.article.on 'change', => @saveArticle()
```

Comment:
What's up with the last_updated?  This was added just below in this file from this PR - https://github.com/artsy/positron/pull/1347

Reply:
Commenting that line out indeed fixes the issue as a listener is no longer registered, but I can't find the root -- meaning, the point in the code that is updating the model every 1 second, triggering this callback.